robot/robot.py

```diff
@@ -89,12 +89,75 @@ def servo_boards(self):
     @property
     def cameras(self):
         """
-        :return: list of available Cameras, can be indexed by serial or by number
+        :return: list of available cameras, can be indexed by serial or by number
         """
         boards = self._update_boards(self.known_cameras, Camera, 'camera')
         self.known_cameras = boards
         return self._dictify_boards(boards)
 
+    @property
+    def _games(self):
+        """
+        :return: list of available GameStates, can be indexed by serial or by number
+        """
+        boards = self._update_boards(self.known_gamestates, GameState, 'game')
+        self.known_gamestates = boards
+        return self._dictify_boards(boards)
+
+    @staticmethod
+    def _single_index(name, list_of_boards):
+        if list_of_boards:
+            return list_of_boards[0]
+        else:
+            raise ValueError("No {}s connected".format(name))
```

Comment:
Minor: As this is raised during property access, is `AttributeError` more usual?